### ISSUE TITLE
What is the validation strategy used when training the S3DIS dataset based on SparseUNet? （semseg-spunet-v1m1-0-base）

### ISSUE THREAD

**Issue by ysy19991107** [Pointcept/Pointcept#406]
Thank you very much for your contribution. I would like to implement semantic segmentation based on the SparseUNet code you shared. 
But while reading the code, I found that both the validation set and the test set are Area_5. What is the reason for this? Is the strategy of using six fold cross validation adopted?

val=dict(
        type=dataset_type,
        split="Area_5",
......

 test=dict(
        type=dataset_type,
        split="Area_5",
......

**Comment by Gofinge** [comment]
Our community usually uses Area 5 for evaluation. For 6-fold validation, you have to change the config to set val and test to each area and use our script to merge the result into one: https://github.com/Pointcept/Pointcept/blob/main/tools/test_s3dis_6fold.py
